asmcomp/emitaux.ml

```diff
@@ -197,8 +197,8 @@ let emit_frames a =
         if Debuginfo.is_none d then 0 else 1
       | Dbg_alloc dbgs ->
         if !Clflags.debug &&
-           List.exists (fun d ->
-             not (Debuginfo.is_none d.Debuginfo.alloc_dbg)) dbgs
+          List.exists (fun d ->
+            not (Debuginfo.is_none d.Debuginfo.alloc_dbg)) dbgs
```

Comment:
I don't like this indentation.

Reply:
Fixed in https://github.com/ocaml-multicore/ocaml/commit/44a3e4db00c40c497e573e47d2aa75defcf45a5a